test/t_cose_sign_verify_test.c

```diff
@@ -8,6 +8,7 @@
  * See BSD-3-Clause license in README.md
  */
 
```

Comment:
There should be a test that tries to use Eddsa without setting the aux buffer to see that the right error is returned.

Reply:
Done. Added both no aux buffer and also a buffer that is too small.